src/libs/Navigation/NavigationRoot.js

```diff
@@ -40,7 +40,7 @@ class NavigationRoot extends Component {
                 ref={navigationRef}
                 linking={linkingConfig}
                 documentTitle={{
-                    formatter: () => 'Expensify.cash',
+                    formatter: () => `${document.title ?? 'Expensify.cash'}`,
```

Comment:
Could we just nix the whole `documentTitle` prop?

Reply:
Then it changes the title to `Report` 😂

Reply:
How about just disabling it?
https://reactnavigation.org/docs/navigation-container/#documenttitle

```jsx
documentTitle={{
    enabled: false,
}}
```

Reply:
That appears to work just fine. I'll change it.